src/util/dom.ts

```diff
@@ -190,8 +190,10 @@ export function pointerCoord(ev: any): Coordinates {
 }
 
 export function hasPointerMoved(threshold: number, startCoord: Coordinates, endCoord: Coordinates) {
-  return startCoord && endCoord &&
-         (Math.abs(startCoord.x - endCoord.x) > threshold || Math.abs(startCoord.y - endCoord.y) > threshold);
+  let deltaX = (startCoord.x - endCoord.x);
```

Comment:
Now it uses the Pythagorean theorem, which is the correct way to calculate distance between two points.